src/utils/standardReport/index.js

```diff
@@ -8,19 +8,23 @@ import { getSelectedReportPeriod } from '../../redux/selectors/reportPeriod/getS
 export const extractRequiredReportParams = reportParams =>
     omitBy(
         {
-            period: reportParams.paramReportingPeriod,
+            period: reportParams.reportingPeriod,
             organisationUnit:
-                reportParams.paramOrganisationUnit ||
-                reportParams.paramParentOrganisationUnit ||
-                reportParams.paramGrandParentOrganisationUnit,
+                reportParams.organisationUnit ||
+                reportParams.parentOrganisationUnit ||
+                reportParams.grandParentOrganisationUnit,
         },
         isFalsy
     )
 
 export const isHtmlReport = report => report.type === reportTypes.HTML
+export const isJasperReportTableReport = report =>
+    report.type === reportTypes.JASPER_REPORT_TABLE
 
 export const getReportParams = report =>
-    report.reportTable ? report.reportTable.reportParams : report.reportParams
+    isJasperReportTableReport(report)
+        ? report.reportTable.reportParams
+        : report.reportParams
```

Comment:
This is the change I mentioned in the description that is not directly related to the API changes. The following was the case:
- I noticed that virtually all reports have report-params both as a direct child and under report-table so it is very unlikely that the current logic was actually correct, we would just end up using the report-table params everywhere.
- I then went to the old struts app to compare some things and I found out that we should use the following distinction instead:
    1. For reports of type HTML & JASPER_JDBC we should use the reportParams directly under `report`. We allow editing them from the reports-app and use them to populate the report-params drop-down.
    2. For reports of type JASPER_REPORT_TABLE we don't allow editing, and we use the report-parameters under report-table.